### ISSUE TITLE
Old Reshader Files

### ISSUE THREAD

**Issue by BloodEvasive** [elishacloud/Silent-Hill-2-Enhancements#447]
Hello, I can't seem to find any resources detailing the old reshader files. The reshader is no longer in the download steps, and I was just curious if these files no longer matter. If these files are no longer required to run the enhanced edition, how do you go about uninstalling them? I've just updated all of my files through the most recent patch, and my game appears to be working. I was just worried about these files potentially causing issues with the updated ones. I've uploaded a screenshot of my files for the game. Thanks!
![Sh2FilesScreenshot](https://user-images.githubusercontent.com/88214987/127718907-bf1000b6-157d-4c98-b2c2-10640501e281.png)

**Comment by Polymega** [comment]
Hi there,

Yes, we've dropped out external need for ReShade as we've now integrated it into the project. This was mostly due to externally using ReShade causing many issues for users. To uninstall it, you'd simply remove the files/folders from the game's directory.

Looking at your screenshot, **remove the following files/folders:**

```
\reshade-shaders\
d3d8.dat
d3d9.dll
d3d9.log
dsound.ini
dsound.log
```

Ultimately you want your game's directory to look like this:
http://enhanced.townofsilenthill.com/SH2/troubleshoot.htm#directoryTree
